src/components/work/overview/Overview.js

```diff
@@ -112,25 +112,27 @@ export function Overview({
                 />
               </Col>
               <Col xs={12}>
-                <Text
-                  type="text3"
-                  className={styles.creators}
-                  skeleton={skeleton}
-                  lines={1}
-                >
-                  {creators.map((c, i) => {
-                    let creatorLink = (
+                {creators.map((c, i) => {
+                  return (
+                    <span>
                       <Link
-                        children={c.name}
+                        children={
```

Comment:
Er der speciel grund til at "children" gives med som prop i stedet for at embedde dem?

Reply:
øh